Index: public_html/index.php
===================================================================
--- public_html/index.php	(revision 8af5a7cbf265dd6201a204056a7a0fedfe53a8c1)
+++ html/index.php	(date 1572124164295)
@@ -1,0 +1,0 @@
Index: public_html/confirm.php
===================================================================
--- public_html/confirm.php	(revision 8af5a7cbf265dd6201a204056a7a0fedfe53a8c1)
+++ html/ip/confirm.php	(date 1572127254465)
@@ -1,0 +1,0 @@
Index: staticIPStuff/thepi-ideal.php
===================================================================
--- staticIPStuff/thepi-ideal.php	(revision 8af5a7cbf265dd6201a204056a7a0fedfe53a8c1)
+++ html/ip/adammilward.co.uk/thepi-ideal.php	(date 1572302455112)
@@ -5,7 +5,7 @@
   echo ' checkIp ';
   $code = rand(10000, 1000000000);
   $ch = curl_init();
-  curl_setopt($ch, CURLOPT_URL, "http://$ip:3142/confirm.php?code=$code");
+  curl_setopt($ch, CURLOPT_URL, "http://$ip:3142/ip/confirm.php?code=$code");
   curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
   curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
   $got = curl_exec($ch);
Index: staticIPStuff/piip.txt
===================================================================
--- staticIPStuff/piip.txt	(revision 8af5a7cbf265dd6201a204056a7a0fedfe53a8c1)
+++ html/ip/adammilward.co.uk/piip.txt	(date 1572126592834)
@@ -1,0 +1,0 @@
Index: public_html/ip.php
===================================================================
--- public_html/ip.php	(revision 8af5a7cbf265dd6201a204056a7a0fedfe53a8c1)
+++ html/ip/ip.php	(date 1572301522522)
@@ -1,0 +1,0 @@
Index: staticIPStuff/thepi.php
===================================================================
--- staticIPStuff/thepi.php	(revision 8af5a7cbf265dd6201a204056a7a0fedfe53a8c1)
+++ html/ip/adammilward.co.uk/thepi.php	(date 1572302427123)
@@ -27,7 +27,7 @@
   return true;
   $code = rand(10000, 1000000000);i
   $ch = curl_init();
-  curl_setopt($ch, CURLOPT_URL, "http://$ip:3142/confirm.php?code=$code");
+  curl_setopt($ch, CURLOPT_URL, "http://$ip:3142/ip/confirm.php?code=$code");
   curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
   curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
   $got = curl_exec($ch);
